packages/components/addon/components/hds/form/utils/describedby.js

```diff
@@ -0,0 +1,12 @@
+import { ID_PREFIX as ERROR_ID_PREFIX } from '../error';
+import { ID_PREFIX as HELPER_TEXT_ID_PREFIX } from '../helper-text';
+
+export function describedBy(element) {
+  let describedBy = [];
+  describedBy.push(`${HELPER_TEXT_ID_PREFIX}${element.id}`);
+  describedBy.push(`${ERROR_ID_PREFIX}${element.id}`);
+  if (element.args.ariaDescribedBy) {
+    describedBy.push(element.args.ariaDescribedBy);
+  }
```

Comment:
@alex-ju I am not sure why you need these lines, we don't have a `@ariaDescribedBy` argument (at the moment, at least)

Reply:
Indeed, we don't. I saw [your comment on accepting an `@id` argument](https://github.com/hashicorp/design-system/pull/285/files#diff-89af5d947bcadceab7186cf644089e49c44401512d8ede02dac0a488d939e74bR44) and based on the spike on `cloud-ui` I'd say we should accommodate that, and also an `@ariaDescribedBy`. We can discuss it in our sync.

Reply:
As discussed, makes totally sense 👍